<!DOCTYPE html>
<html lang='en'>
  <head>
    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PK63DK');</script>
    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type'>
    <meta charset='utf-8'>
    <meta content='width=device-width, initial-scale=1, shrink-to-fit=no' name='viewport'>
    <link crossorigin='anonymous' href='https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css' integrity='sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB' rel='stylesheet'>
    <link href='/styles.css' rel='stylesheet'>
    <title>How cells 'hide' potentially dangerous DNA segments</title>
    <script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script><script>var googletag=googletag ||{}; googletag.cmd=googletag.cmd || [];</script><script>googletag.cmd.push(function(){googletag.defineSlot('\/1010898\/sgforums_interstitial', [ 1.0 , 1.0], 'div-gpt-ad-1528181032761-0').setTargeting('forum', ['2258']).setTargeting('topic', ['355614']).addService(googletag.pubads()); googletag.defineSlot('\/1010898\/sgforums_leaderboard', [[ 728.0 , 90.0], [ 970.0 , 250.0]], 'div-gpt-ad-1528181032761-1').setTargeting('forum', ['2258']).setTargeting('topic', ['355614']).addService(googletag.pubads()); googletag.defineSlot('\/1010898\/sgforums_mpu', [ 300.0 , 250.0], 'div-gpt-ad-1528181032761-2').setTargeting('forum', ['2258']).setTargeting('topic', ['355614']).addService(googletag.pubads()); googletag.defineSlot('\/1010898\/sgforums_supersky', [ 160.0 , 600.0], 'div-gpt-ad-1528181032761-3').setTargeting('forum', ['2258']).setTargeting('topic', ['355614']).addService(googletag.pubads()); googletag.pubads().enableSingleRequest(); googletag.enableServices();});</script>
  </head>
  <body>
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PK63DK"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <div class='container'>
      <a href='/'>Home</a>
      >
      <a href='/forums/2258/'>Science Forum</a>
      <h1>How cells 'hide' potentially dangerous DNA segments</h1>
      <nav class='pagination'>
      </nav>
      <ul class='list-group posts'>
        <li class='list-group-item post'>
          <div class='row'>
            <div class='col-md-2 meta'>
              <div class='author'>Bangulzai</div>
              <time class='date' datetime='2009-04-12T02:16:13+00:00'>12 Apr 09, 02:16</time>
            </div>
            <div class='col-md body'>
              <p><strong><span style="color: #3366ff;">How cells 'hide' potentially dangerous DNA segments</span></strong></p>
              <p><em><span style="color: #3366ff;"><img src="http://www.labspaces.net/images/news/914335___dna__.jpg" alt=""></span></em></p>
              <p><span style="color: #3366ff;"><em>Friday, April 10, 2009</em><br><br>The DNA in the 23 pairs of chromosomes in each of the billions of cells of the human body is so tightly packed that it would measure six feet in length if stretched end to end. A genome of this size can squeeze into a cell's tiny nucleus because it is compressed into highly condensed chromatin fibers by proteins called histones. <br><br>All chromatin in the cell nucleus represents a massive condensation of the genetic material. But a portion of it might well be called super-condensed; it forms a kind of chromatin called heterochromatin. The genes contained within these portions of the genome are effectively "silenced" because they cannot be accessed by the cell's DNA-activating machinery. These "hidden" parts of the genome also include highly repetitive, gene-poor, regions. Some of these, if unpacked, would set loose DNA sequences that act like parasites – able to jump around to other areas, sometimes randomly, unleashing genetic chaos. <br><br>To assemble heterochromatin, numerous molecules participate in an elaborate series of maneuvers that have gradually come to light. "But scientists have been a little hazy on the initial steps and requirements that get this process going," says Professor Leemor Joshua-Tor, Ph.D., of Cold Spring Harbor Laboratory (CSHL). She and her research team have now brought this process into sharper focus by identifying a critical requirement for heterochromatin to be established in the nucleus. <br><br>In a report that appears online on April 9th in the journal Molecular Cell, they show that the assembly of heterochromatin depends on the strength with which a protein called Chp1 binds to a specific target site located on a histone protein that has attached to the double helix. <br><br>RNAi's role in heterochromatin formation <br><br>In a typical chromosome -- roughly the shape of a bow-tie – heterochromatin is concentrated at (and supports the structure of) the centromere, the bow's central "knot." Since centromeres are crucial for genomic integrity, aberrant heterochromatin formation can result in genetic anomalies and diseases such as cancer. <br><br>Heterochromatin depends on the density of chemical "marks" that are added to histones at specific locations on their tail ends. These marks consist of methyl groups that get attached to the tail of one of the histones, histone H3, at a specific spot – the ninth residue, which happens to be the amino acid lysine (K). This mark is therefore called H3K9me. <br><br>In fission yeast, a model system used for studying heterochromatin mechanisms because of its comparative simplicity, the precise pattern of methylation depends on a process called RNA interference, or RNAi. It involves a host of players: the enzyme that copies DNA into RNA, which is diced into short segments called short interfering RNAs, or siRNAs; a part of the RNAi machinery known as the RITS complex; and various enzymes that are able to alter the configuration of chromatin. <br><br>A debate over recruitment <br><br>"In trying to understand the interplay between each of these components," explains Joshua-Tor, "there has been a longstanding debate over how the RITS complex initially gets recruited to the regions at the centromere that are destined to become heterochromatin." <br><br>At first, scientists thought that the siRNA molecules acted as guides to recruit RITS. After latching on to the chromatin, RITS was in turn thought to recruit the other components: the enzyme that adds the methyl marks; and the protein Chp1, which acts as a molecular "velcro" to keep the RITS complex firmly attached to the methyl-decorated chromatin. <br><br>The importance of binding strength <br><br>The CSHL team in collaboration with the team of Janet Partridge at St. Jude's Children's Research Hospital, Memphis, has now found that siRNAs cannot do the job of heterochromatin assembly by themselves. Rather, the siRNA-guided interaction relies on the strength with which the velcro-like protein Chp1 binds to methylated chromatin. <br><br>"We found that a part of Chp1 known as the chromodomain binds with high affinity (or strength) to methylated chromatin," explains Thomas Schalch, Ph.D., a postdoctoral researcher in the Joshua-Tor lab who led the current study. By teasing apart the origin of this unique affinity, the CSHL team stumbled across Chp1 mutants that would produce siRNAs but could not assemble heterochromatin. <br><br>"These results lead us to think that the tight interaction between RITS and the methyl marks is a requirement at least as important as the availability of siRNA," Schalch says. <br><br>A crystal structure provides the answer <br><br>The team wanted to know the exact points of contact between Chp1 and its target and how these interactions contributed to the strength of binding. For these answers, Schalch and the CSHL team made use of Joshua-Tor's expertise in X-ray crystallography – the science of determining the exact position of atoms and bonds within a molecule by creating a crystal, which is then probed with powerful x-rays. <br><br>Disrupting each point of interaction between Chp1 and its target by engineering various mutations into the Chp1 protein decreased the strength of binding to different levels. Even a five-fold decrease in binding strength prevented the mutation-bearing cells from assembling new heterochromatin, even though some of the mutants were able to generate siRNAs. <br><br>This work reveals that siRNAs cannot by themselves establish heterochromatin when Chp1's binding to H3K9me – the methylated chromatin – is impaired. Whether this mechanism of heterochromatin assembly discovered in yeast also occurs in mammalian cells is unclear at this point, according to Schalch, as a mammalian equivalent to Chp1 has not been found. "But we've now identified a fingerprint for the high-affinity interaction between a chromatin-binding protein and its 'marked', or methylated, target," says Schalch. "This may help us identify Chp1-like proteins in mammalian cells." <br><br>### <br><br>Cold Spring Harbor Laboratory</span></p>
              <p><span style="color: #3366ff;">Thanks to Cold Spring Harbor Laboratory for this article.</span></p>
            </div>
          </div>
        </li>
        <li class='list-group-item'>
          <div id='div-gpt-ad-1528181032761-2' style='height:250px; width:300px;'><script>googletag.cmd.push(function(){googletag.display('div-gpt-ad-1528181032761-2');});</script></div>
        </li>
      </ul>
      <nav class='pagination'>
      </nav>
      <footer>
        <nav class='nav'>
          <a class='nav-link' href='/'>Home</a>
          <a class='nav-link' href='/privacy/'>Privacy</a>
          <a class='nav-link' href='/terms/'>Terms</a>
        </nav>
      </footer>
    </div>
    <div id='div-gpt-ad-1528181032761-0' style='height:1px; width:1px;'><script>googletag.cmd.push(function(){googletag.display('div-gpt-ad-1528181032761-0');});</script></div>
  </body>
</html>
