<!DOCTYPE html>
<html lang='en'>
  <head>
    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PK63DK');</script>
    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type'>
    <meta charset='utf-8'>
    <meta content='width=device-width, initial-scale=1, shrink-to-fit=no' name='viewport'>
    <link crossorigin='anonymous' href='https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/css/bootstrap.min.css' integrity='sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB' rel='stylesheet'>
    <link href='/styles.css' rel='stylesheet'>
    <title>The evolution of genomes and language</title>
    <script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script><script>var googletag=googletag ||{}; googletag.cmd=googletag.cmd || [];</script><script>googletag.cmd.push(function(){googletag.defineSlot('\/1010898\/sgforums_interstitial', [ 1.0 , 1.0], 'div-gpt-ad-1528181032761-0').setTargeting('forum', ['2258']).setTargeting('topic', ['409701']).addService(googletag.pubads()); googletag.defineSlot('\/1010898\/sgforums_leaderboard', [[ 728.0 , 90.0], [ 970.0 , 250.0]], 'div-gpt-ad-1528181032761-1').setTargeting('forum', ['2258']).setTargeting('topic', ['409701']).addService(googletag.pubads()); googletag.defineSlot('\/1010898\/sgforums_mpu', [ 300.0 , 250.0], 'div-gpt-ad-1528181032761-2').setTargeting('forum', ['2258']).setTargeting('topic', ['409701']).addService(googletag.pubads()); googletag.defineSlot('\/1010898\/sgforums_supersky', [ 160.0 , 600.0], 'div-gpt-ad-1528181032761-3').setTargeting('forum', ['2258']).setTargeting('topic', ['409701']).addService(googletag.pubads()); googletag.pubads().enableSingleRequest(); googletag.enableServices();});</script>
  </head>
  <body>
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-PK63DK"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    <div class='container'>
      <a href='/'>Home</a>
      >
      <a href='/forums/2258/'>Science Forum</a>
      <h1>The evolution of genomes and language</h1>
      <nav class='pagination'>
      </nav>
      <ul class='list-group posts'>
        <li class='list-group-item post'>
          <div class='row'>
            <div class='col-md-2 meta'>
              <div class='author'>BanguIzai</div>
              <time class='date' datetime='2010-08-31T23:26:56+00:00'>31 Aug 10, 23:26</time>
            </div>
            <div class='col-md body'>
              <p><span style="font-family: Arial, sans-serif; color: #212121; font-size: x-large;"><span style="font-size: 19px; line-height: normal;  "><strong><span style="font-weight: normal; font-size: 15px;">
              </span></strong></span></span></p><div class="front-matter-section" style="width: inherit; margin-top: 0em; margin-bottom: 1.2em; word-wrap: break-word;"><strong>
              <div class="fm-title" style="margin-top: 16px; font-size: 19px; font-weight: bold;">The evolution of genomes and language</div>
              <div class="contrib-group fm-author" style="font-size: 17px; margin-top: 16px !important; margin-bottom: 16px !important;">Hong-Yu Zhang<sup style="font-size: 0.8em; line-height: 0.8em;">1</sup></div>
              <div class="fm-affl" style="font-weight: normal; color: #5e5e5e !important; font-size: 12px; margin: 0px !important;"><sup style="font-size: 0.8em; line-height: 0.8em;">1</sup>Hong-Yu Zhang is at the Shandong Provincial Research Center for Bioinformatic Engineering and Technique, Shandong University of Technology, Zibo, People's Republic of China. <span class="email-label">Email:</span><span class="e_id513471">zhanghy@sdut.edu.cn</span></div>
              </strong></div><strong>
              <div id="__bodyid531563" class="sec" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: inherit;">
              <div id="__bodyid531563titletitle" class="head1 section-title" style="width: inherit; margin-top: 0em; margin-bottom: 0.909em; word-wrap: break-word; padding-bottom: 0.1em; padding-top: 0.1em; font-size: 17px; background-color: #0072a8; color: white; font-family: Verdana, sans-serif; font-weight: bold; text-align: center; text-transform: none; border: 1pt none #0072a8;">
              <div class="other-sections" style="float: right; padding-top: 0px; padding-right: 0.25em; padding-bottom: 0px; padding-left: 0.25em; position: relative;">
              <ul class="noext-menu" style="list-style-type: none; list-style-position: initial; list-style-image: initial; line-height: 1.2em; text-align: left; padding: 0px; margin: 0px;">
              <li> Other Sectionsâ–¼</li>
              </ul>
              </div>
              <div> </div>
              </div>
              <div id="__bodyid531563content" class="section-content" style="width: inherit; margin-top: 0px; margin-bottom: 1.2em; word-wrap: break-word; line-height: 22px;">
              <div id="__pid511036" class="p p-first" style="margin-top: 0px; margin-bottom: 1em; word-wrap: break-word; width: 550px;">Since Charles Darwin published <em>The Origin of Species</em> in 1859, his theory of evolution has been a central axiom for the biological sciences. Scientists have devoted considerable effort to unravelling the evolutionary mechanisms of biological, and other, systems. Their work—in particular the discovery of DNA as the carrier of hereditary information—has led to the now nearly universally accepted idea that evolution causes the increasing complexity of most biological systems. However, although the basic mechanisms of evolution—mutation and selection—are clear, accumulating data suggest that the means to increase complexity have also become more complex. This begs the question how such mechanisms have evolved over time.</div>
              <div id="__pid511048" class="p" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: 550px;">Thanks to rapid progress in the biological sciences—in particular in the field of genomics—and the information sciences including linguistics, we have amassed an enormous amount of data on genomes and biological evolutionary mechanisms. These are providing an unprecedented opportunity to investigate the possible transformation of evolutionary strategies of evolving systems, such as genomes and languages. In fact, the information contained in genomes has long been compared with languages, and many linguistic methodologies are now used to analyse genomes (Searls, 2002). If we compare the evolution of genomes and language, we serendipitously find that both systems have undergone similar strategic shifts to attain increasing complexity, which suggests that this is an intrinsic property of many evolving systems, not just biological ones.</div>
              <div id="__pid511068" class="p" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: 550px;">During the primary stage in the evolution of both genomes and languages, increasing complexity was achieved mainly by increasing the number of basic information-carrying elements: nouns and verbs in language, and genes in biology. The Chinese language is a good example to use in this context, because of its history of more than 6,000 years and its continuing evolution. The early Chinese written language, called Oracle, consisted of a few thousand characters, which gradually increased to include more than 47,000 individual characters, as the ancient Chinese continually invented characters to refine their ability to describe their environment (<span style="text-decoration: underline;">Table 1</span>; Ji, 1989). For example, more than 70 characters described horses on the basis of their colour, age and gender, and more than 80 characters described their behaviour (Chen, 1936). According to a statistical analysis of 800 million ancient characters, one would need to know about 22,000 characters to attain 99.99% coverage of ancient Chinese (Zhang, 2004). The same phenomenon can be observed in biological evolution. Simple and early organisms, such as prokaryotes, manage to survive and proliferate with a few thousand genes, whereas the number of genes in higher organisms is about a magnitude higher—some plants, such as maize, have more than 50,000 genes (Messing <em>et al</em>, 2004).</div>
              <div class="canvas-table-ref-outer" style="margin-top: 1.3125em; margin-right: 0.5em; margin-bottom: 1.3125em; margin-left: 0.5em; border-top-color: #999999; border-right-color: #999999; border-bottom-color: #aaaaaa; border-left-color: #aaaaaa; border-width: 1px; border-style: solid;">
              <div class="canvas-table-ref-inner" style="border: 3px solid #f0f0f0;">
              <table class="thumb-caption" style="clear: both; width: 528px;" border="0" cellspacing="0" cellpadding="0">
              <tbody>
              <tr>
              <td class="thumb-cell" style="width: 110px;"><a id="t1" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1525159/table/t1/">
              <div class="thumb-ph" style="padding-top: 0.5em; padding-right: 0px; padding-bottom: 0.5em; padding-left: 0.5em;"><img class="icon-reflink" style="border: 1px solid initial;" title="Table 1" src="http://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/table-icon.gif" alt="Table 1"></div>
              </a></td>
              <td class="caption-cell" style="text-align: left;">
              <div class="caption-ph" style="padding: 0.5em;"><strong>Table 1</strong>
              <div class="figure-table-caption-in-article" style="line-height: 1.4; font-size: 13px; color: #5e5e5e;"><span>Evolution of Chinese characters (a complete list of sources is listed in Ji (1989))</span></div>
              </div>
              </td>
              </tr>
              </tbody>
              </table>
              </div>
              </div>
              <div id="__pid506617" class="p" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: 550px;">In the second stage of their evolution, both systems began to rearrange existing elements into new combinations to increase complexity further. Interestingly, this leads to a decrease in the quantity of elements. For example, modern Chinese uses considerably fewer characters than did the ancient language: only 4,600 characters are now needed to attain 99.99% coverage (Zhang, 1997). However, modern Chinese is definitely more powerful than its ancient predecessor, because it is able to describe a much more complex world. The most frequently used 3,500 characters—covering about 99.87% of modern Chinese—can be combined to form more than 70,000 words (Zhang, 1997), which include the meanings of most ancient characters. Genomes have undergone a similar evolutionary shift. Mammals—such as <em>Homo sapiens</em> and the mouse—have about 25,000 genes (International Human Genome Sequencing Consortium, 2004; Guénet, 2005), which is much fewer than some plants: for example, rice has 43,000 genes (Paterson <em>et al</em>, 2005) and maize has 59,000 (Messing <em>et al</em>, 2004). The greater complexity of mammals is explained partly by the recombination of existing genes, through mechanisms such as alternative splicing (Johnson <em>et al</em>, 2003) and tandem chimerism (Parra <em>et al</em>, 2006). In fact, mammals depend much more on gene recombination to achieve higher complexity than do plants (Messing, 2001).</div>
              <div id="__pid835042" class="p" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: 550px;">The third stage of evolution witnesses the arrival of ‘virtual', or modifying, elements, which have an important role as regulatory components. In language, adverbs, auxiliary words, prepositions and conjunctions are all virtual elements, whereas nouns and verbs are the main carriers of information. In fact, the five most frequently used characters in modern Chinese contain two ‘empty' elements (<span style="text-decoration: underline;">Fig 1</span>). Although virtual words were quite rare in Oracle, they are much more frequent in modern Chinese. Similarly, non-coding RNA has a virtual role in genomes, whereas protein-coding genes are the main carriers of information. In mammals, it is RNA, not protein, that mainly controls gene activity (Mattick, 2004), which also helps to explain the unexpectedly low number of protein-coding genes in mammalian genomes.</div>
              <div class="canvas-figure-ref-outer" style="margin-top: 1.3125em; margin-right: 0.5em; margin-bottom: 1.3125em; margin-left: 0.5em; border-top-color: #999999; border-right-color: #999999; border-bottom-color: #aaaaaa; border-left-color: #aaaaaa; border-width: 1px; border-style: solid;">
              <div class="canvas-figure-ref-inner" style="border: 3px solid #f0f0f0;">
              <table class="thumb-caption" style="clear: both; width: 528px;" border="0" cellspacing="0" cellpadding="0">
              <tbody>
              <tr>
              <td class="thumb-cell" style="width: 110px;"><a id="f1" class="icon-reflink figpopup" style="position: relative; white-space: nowrap; cursor: pointer; display: block;" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1525159/figure/f1/">
              <div class="thumb-ph" style="padding-top: 0.5em; padding-right: 0px; padding-bottom: 0.5em; padding-left: 0.5em; position: relative; white-space: nowrap; cursor: pointer;">
              <div class="small-thumb-canvas" style="max-width: 104px; white-space: normal; word-wrap: break-word; border: 1px solid #444444;">
              <div class="small-thumb-canvas-1" style="border: 1px solid transparent;"><img class="icon-reflink small-thumb" style="width: 100px; border: 1px solid #444444;" title="Figure 1" src="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1525159/bin/7400756-f1.gif" alt="Figure 1"></div>
              </div>
              </div>
              </a></td>
              <td class="caption-cell" style="text-align: left;">
              <div class="caption-ph" style="padding: 0.5em;"><strong>Figure 1</strong>
              <div class="figure-table-caption-in-article" style="line-height: 1.4; font-size: 13px; color: #5e5e5e;"><span>Examples of ‘virtual' elements in modern Chinese. The symbol on the left means ‘of' and the symbol on the right conveys a form of past tense (National Council of Language and Writing, 1989).</span></div>
              </div>
              </td>
              </tr>
              </tbody>
              </table>
              </div>
              </div>
              <div id="__pid835073" class="p p-last" style="margin-top: 0.3125em; margin-bottom: 0px; word-wrap: break-word; width: 550px;">In summary, the evolution of both genomes and language has led to increased complexity, but has also developed novel strategies to enhance this process further. Thus, evolution has undergone several paradigm shifts: from increasing the number of basic elements to shuffling existing building blocks into new combinations, and finally to creating virtual regulatory elements. These strategic shifts improve the overall complexity and performance of the systems at a relatively low cost: the ability to combine elements reduces the number of building blocks needed, and virtual elements simplify regulation. Using RNA rather than proteins as regulators circumvents protein translation, and the average length of such RNA signals—22 nucleotides—is almost two orders of magnitude shorter than that required to encode an average protein (Mattick, 2004). It is therefore reasonable to propose that this “evolution of evolutionary strategies” is an intrinsic property of evolving systems. It will be a great challenge to provide further proof—for example, by establishing a model to simulate such higher-level evolution—especially <em>in silico</em>.</div>
              </div>
              </div>
              <div id="__ackid551885" class="sec" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: inherit;">
              <div id="__ackid551885titletitle" class="head1 section-title" style="width: inherit; margin-top: 0em; margin-bottom: 0.909em; word-wrap: break-word; padding-bottom: 0.1em; padding-top: 0.1em; font-size: 17px; background-color: #0072a8; color: white; font-family: Verdana, sans-serif; font-weight: bold; text-align: center; text-transform: none; border: 1pt none #0072a8;">
              <div>Acknowledgments</div>
              </div>
              <div id="__ackid551885content" class="section-content" style="width: inherit; margin-top: 0px; margin-bottom: 1.2em; word-wrap: break-word; line-height: 22px;">
              <div class="sec" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: inherit;">
              <div id="__pid551886" class="p" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: inherit;">This study was supported by the National Basic Research Program of China and the National Natural Science Foundation of China.</div>
              </div>
              </div>
              </div>
              <div id="__ref-listid551890" class="sec" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: inherit;">
              <div id="__ref-listid551890titletitle" class="head1 section-title" style="width: inherit; margin-top: 0em; margin-bottom: 0.909em; word-wrap: break-word; padding-bottom: 0.1em; padding-top: 0.1em; font-size: 17px; background-color: #0072a8; color: white; font-family: Verdana, sans-serif; font-weight: bold; text-align: center; text-transform: none; border: 1pt none #0072a8;">
              <div class="other-sections" style="float: right; padding-top: 0px; padding-right: 0.25em; padding-bottom: 0px; padding-left: 0.25em; position: relative;">
              <ul class="noext-menu" style="list-style-type: none; list-style-position: initial; list-style-image: initial; line-height: 1.2em; text-align: left; padding: 0px; margin: 0px;">
              <li> Other Sectionsâ–¼</li>
              </ul>
              </div>
              <div>References</div>
              </div>
              <div id="__ref-listid551890content" class="section-content" style="width: inherit; margin-top: 0px; margin-bottom: 1.2em; word-wrap: break-word; line-height: 22px;">
              <div class="sec" style="margin-top: 0.3125em; margin-bottom: 1em; word-wrap: break-word; width: inherit;">
              <div id="reference-list" class="back-matter-section" style="width: inherit; margin-top: 0em; margin-bottom: 1.2em; word-wrap: break-word; font-size: 12px;">
              <ul class="back-ref-list-1st-line-shifted" style="padding: 0px;">
              <li id="b1" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid551896" class="citation">Chen TJ (1936) <span class="ref-journal" style="font-style: italic;">Kangxi Dictionary</span>. Shanghai, China: World Press.</span></li>
              <li id="b2" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid551921" class="citation">Guénet JL (2005) <span class="ref-title">The mouse genome</span>. <span class="ref-journal" style="font-style: italic;">Genome Res</span> <span class="ref-vol">15</span>: 1729–1740 <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b3" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid551966" class="citation">International Human Genome Sequencing Consortium (2004) <span class="ref-title">Finishing the euchromatic sequence of the human genome</span>. <span class="ref-journal" style="font-style: italic;">Nature</span> <span class="ref-vol">431</span>: 931–945 <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b4" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid552004" class="citation">Ji XL (1989) <span class="ref-journal" style="font-style: italic;">Encyclopedia of China: Language and Character</span>. Beijing, China: Encyclopedia of China Publishing House.</span></li>
              <li id="b5" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid552029" class="citation">Johnson JM, Castle J, Garrett-Engele P, Kan Z, Loerch PM, Armour CD, Santos R, Schadt EE, Stoughton R, Shoemaker DD (2003) <span class="ref-title">Genome-wide survey of human alternative pre-mRNA splicing with exon junction microarrays</span>. <span class="ref-journal" style="font-style: italic;">Science</span> <span class="ref-vol">302</span>: 2141–2144 <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b6" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid552146" class="citation">Mattick JS (2004) <span class="ref-title">RNA regulation: a new genetics?</span> <span class="ref-journal" style="font-style: italic;">Nat Rev Genet</span> <span class="ref-vol">5</span>: 316–323 <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b7" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid552191" class="citation">Messing J (2001) <span class="ref-title">Do plants have more genes than humans?</span> <span class="ref-journal" style="font-style: italic;">Trends Plant Sci</span> <span class="ref-vol">6</span>: 195–196 <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b8" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid552236" class="citation">Messing J et al. . (2004) <span class="ref-title">Sequence composition and genome organization of maize</span>. <span class="ref-journal" style="font-style: italic;">Proc Natl Acad Sci USA</span> <span class="ref-vol">101</span>: 14349–14354 <span style="white-space: nowrap;">[PMC free article]</span> <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b9" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid552284" class="citation">National Council of Language and Writing (1989) <span class="ref-journal" style="font-style: italic;">Frequency Statistics of Frequently Used Modern Chinese Characters</span>. Beijing, China: Yuwen Press.</span></li>
              <li id="b10" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid552303" class="citation">Parra G, Reymond A, Dabbouseh N, Dermitzakis ET, Castelo R, Thomson TM, Antonarakis SE, Guigó R (2006) <span class="ref-title">Tandem chimerism as a means to increase protein complexity in the human genome</span>. <span class="ref-journal" style="font-style: italic;">Genome Res</span> <span class="ref-vol">16</span>: 37–44 <span style="white-space: nowrap;">[PMC free article]</span> <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b11" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid843469" class="citation">Paterson AH, Freeling M, Sasaki T (2005) <span class="ref-title">Grains of knowledge: genomics of model cereals</span>.<span class="ref-journal" style="font-style: italic;">Genome Res</span> <span class="ref-vol">15</span>: 1643–1650 <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b12" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid843530" class="citation">Searls DB (2002) <span class="ref-title">The language of genes</span>. <span class="ref-journal" style="font-style: italic;">Nature</span> <span class="ref-vol">420</span>: 211–217 <span style="white-space: nowrap;">[PubMed]</span></span></li>
              <li id="b13" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid843574" class="citation">Zhang K (1997) <span class="ref-title">Statistical analysis on basic characters constituting Chinese words</span>. <span class="ref-journal" style="font-style: italic;">Yuyan Jiaoxue Yu Yanjiu</span> <span class="ref-vol">1997</span>: 43–51 [In Chinese]</span></li>
              <li id="b14" style="margin-top: 1.4em; line-height: 1.4em; list-style-type: none; margin-left: 2em; text-indent: -1.5em;"><span id="__citationid843616" class="citation">Zhang ZC (2004) <span class="ref-title">Large-scale character-usage statistical analysis on Chinese ancient characters</span>. In <span class="ref-journal" style="font-style: italic;">Proceedings of Third Symposium on Databases of Chinese Writing and History</span>, pp 107–119. Beijing, China: National Committee for Programming Publishing House and National League of Ancient Book Publishing House.</span></li>
              </ul>
              </div>
              </div>
              </div>
              </div>
              <hr style="color: #0072a8;">
              <div class="footer-section sans90" style="width: inherit; margin-top: 0px; margin-bottom: 1em; word-wrap: break-word; font-family: Arial, sans-serif; font-size: 14px; margin-left: 1em; margin-right: 1em; text-align: center;">
              <div>Articles from <span class="acknowledgment-journal-title" style="font-style: italic;">EMBO Reports</span> are provided here courtesy of</div>
              <div><strong class="src">The European Molecular Biology Organization</strong></div>
              </div>
              </strong><p></p>
            </div>
          </div>
        </li>
        <li class='list-group-item'>
          <div id='div-gpt-ad-1528181032761-2' style='height:250px; width:300px;'><script>googletag.cmd.push(function(){googletag.display('div-gpt-ad-1528181032761-2');});</script></div>
        </li>
      </ul>
      <nav class='pagination'>
      </nav>
      <footer>
        <nav class='nav'>
          <a class='nav-link' href='/'>Home</a>
          <a class='nav-link' href='/privacy/'>Privacy</a>
          <a class='nav-link' href='/terms/'>Terms</a>
        </nav>
      </footer>
    </div>
    <div id='div-gpt-ad-1528181032761-0' style='height:1px; width:1px;'><script>googletag.cmd.push(function(){googletag.display('div-gpt-ad-1528181032761-0');});</script></div>
  </body>
</html>
